providers/d7-viber-chat/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@novu/d7-viber-chat",
+  "version": "0.16.3",
```

Comment:
The version needs to be updated to the .20 something like the rest of the providers